From 864ac2f1108a84fb3bfded21177229f34ac87e05 Mon Sep 17 00:00:00 2001
From: Sergei Trofimovich <slyich@gmail.com>
Date: Sun, 20 Feb 2022 18:46:37 +0000
Subject: [PATCH] Makefile: use /usr/bin/env bash instead of /bin/bash

On NixOS of some BSDs bash might not be present in /bin.
For me it's in

    $ which bash
    /run/current-system/sw/bin/bash

and build fails as:

    $ make
    make: /bin/bash: No such file or directory

The change uses slightly more stable '/usr/bin/env bash' indirection.

Signed-off-by: Sergei Trofimovich <slyich@gmail.com>
---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index 098f0bd..cfa2229 100644
--- a/Makefile
+++ b/Makefile
@@ -1,7 +1,7 @@
 # Copyright (c) 2022 by Alejandro Colomar <alx.manpages@gmail.com>
 # SPDX-License-Identifier: GPL-2.0
 
-SHELL := /bin/bash -Eeuo pipefail
+SHELL := /usr/bin/env bash -Eeuo pipefail
 
 MAKEFLAGS += --no-builtin-rules
 MAKEFLAGS += --no-builtin-variables
-- 
2.35.1

